src/mmw/apps/modeling/geoprocessing.py

```diff
@@ -47,11 +48,32 @@
 }
 
 
-def histogram(polygons):
+def histogram_start(polygons):
     """
-    Send a list of strings containing a GeoJSON Polygons and/or
-    MultiPolygons to the datahub code, and get back a histogram of
-    the NLCD x SOIL pairs present in that area.
+    Together, histogram_start and histogram_finish implement a
+    function which takes a list of polygons or multipolygons as input,
+    and returns histograms of the NLCD x soil pairs that occur in each
+    shape.
+
+    This is the top-half of the function.
+    """
+    host = settings.GEOP['host']
+    port = settings.GEOP['port']
+    path = settings.GEOP['path']
+    request = settings.GEOP['request'].copy()
+    request['input']['geometry'] = polygons
+    url = "http://%s:%s%s" % (host, port, path)
+
+    data = requests.post(url, data=json.dumps(request)).json()
```

Comment:
Some example usage of the StatsD client is in the [Boto mailer backend](https://github.com/WikiWatershed/model-my-watershed/blob/53c325afa9b0ba209ce9ca7c29ccf2b3eec8ee3a/src/mmw/apps/core/mail/backends/boto_ses_mailer.py). More detail into the available data types can be found [here](http://statsd.readthedocs.org/en/latest/types.html).